src/duckling/game/drawable/shape-drawable.ts

```diff
@@ -0,0 +1,38 @@
+import {Drawable, DrawableType} from './drawable';
+import {Shape} from './shape';
+
+export interface ShapeDrawable extends Drawable {
+    shape: Shape;
+}
+
+export var defaultShapeDrawable : ShapeDrawable = {
```

Comment:
Whenever I implement the defaults for the more specific interfaces that extend (as seen in ContainerDrawable, Rectangle, Circle) I need to retype all the properties that are general to the extended interface. So properties in defaultDrawable become repeated here (such as scale, renderPriority, positionOffset, etc). Can I get around this?